configure-time-zone.md

```diff
@@ -109,3 +95,11 @@ In this example, no matter how you adjust the value of the time zone, the value
 >
 > - Time zone is involved during the conversion of the value of Timestamp and Datetime, which is handled based on the current `time_zone` of the session.
 > - For data migration, you need to pay special attention to the time zone setting of the primary database and the secondary database.
+> - To get correct timestamps it is strongly recommended to setup a reliable clock with NTP or PTP. See [system configuration check: NTP](/check-before-deployment.md#check-and-install-the-ntp-service) for how to check this.
+> - When using a timezone that observes daylight saving this might lead to ambigious timestamps and/or nonexistent timestamps, especially if calculations are done with the timestamp.
+> - MySQL uses `mysql_tzinfo_to_sql` to convert the timezone database of the OS to tables in the database. This isn't used or needed with TiDB as TiDB relies on timezone data that's part of the Go programming language.
```

Comment:
Do we need to also say that depending on the environment it will use the time zone info files from the OS, as golang's time package says "use the file <time_zone> in the system timezone directory"?

Reply:
This is quite a big advantage. Where [`CONVERT_TZ`](https://dev.mysql.com/doc/refman/8.4/en/date-and-time-functions.html#function_convert-tz) in MySQL might return NULL because of missing timezone data that never happens with TiDB.

In addition to this the data in TiDB doesn't need to get updated when the [Olson database](https://en.wikipedia.org/wiki/Tz_database) gets updated. Note that MySQL doesn't record a version or timestamp, which makes it impossible to see if the data is outdated and needs to be loaded again. See also https://bugs.mysql.com/bug.php?id=60675